src/libraries/System.Security.Cryptography.Pkcs/tests/Pkcs12/Pkcs12SafeBagTests.cs

```diff
@@ -67,10 +67,6 @@ public static void GetBagIdIsFactory()
             Assert.NotSame(firstCall, secondCall);
             Assert.Equal(Oids.Aes192, firstCall.Value);
             Assert.Equal(firstCall.Value, secondCall.Value);
-
-            secondCall.Value = Oids.Cms3DesWrap;
```

Comment:
Presumably this was testing what we are trying to fix, that `GetBagId` returns a defensive copy. I think the `NotSame` covers that well enough for now, and it'll get further fixed when / if this becomes a place defensive copies are removed.